components/linear/linear.app.mjs

```diff
@@ -11,5 +11,43 @@ export default {
         ...options,
       };
     },
+    async getOwnUserInfo() {
+      const {
+        avatarUrl, displayName,
+      } = await this.client().viewer;
+      return {
+        avatarUrl,
+        displayName,
+      };
+    },
+  },
+  propDefinitions: {
+    ...linearApp.propDefinitions,
+    createAs: {
```

Comment:
This should be an optional prop

Reply:
Alright - changed it to optional (default value was already set as "me" = authenticated user)